### DISCUSSION TITLE
Fixing the MMR & smurfing issue for good.

### DISCUSSION THREAD

**Discussion by Aikirob** [ValveSoftware/Dota2-Gameplay#22286] [Ideas]
a very simple change could end the horrid match-ups in matchmaking games.

give players mmr based on their KDA AND games won instead of just games won.

if you Win the game, you still get the +25 mmr or whatever it is, but have extra points added or subtracted based on your K/D/A, for example

player wins match +25mmr, player got 20/5/10 score during the match, give +1 mmr per kill, -1 mmr per death and +0.5 mmr for an assist, that gives this player an increase of +45 for an exceptional game on their part. 

On the flipside we have a feeder who loses the match with 0/10/4, they get -25mmr or however you currently calculate that, -10mmr for dying 10 times and +2mmr for assists, that's -33mmr for quite a poor game played, this also incentivises good team work as assist mmr is valuable!

This way if people smurf in lower ranks they rank up faster and can't troll the new players for an extended period of time.

People who ruin games or play terribly will be put where they belong, with players their own skill level and people who deliberately feed to abuse the system go exactly where they belong.

This very simple change will fix the matchmaking system indefinitely.
Please use this formula.

**Comment by z3niths** [comment]
Well, nobody will play initiator or support since it's led to low KDA :P
Smurf or Booster, when playing another player account, can also gain a lot of MMR quickly with that system.
The length of the game is also one of the variables.

The best would be, that the system can determine `performance` (hero-position performance compared to the average hero-position-rank-performance) based on game time, and use it as a bonus which I believe they already did that but only jump and effect when they sum up for long period and re-calibrate every season to make sure that the booster won't work if they didn't play the account long enough.

**Reply by Aikirob** [reply]
That's a good thing, it means you have to be more calculated and not jump in blindly.

**Comment by commentator24** [comment]
your suggestion similar to #21934 KDA wont work because ppl will just prolong the game instead of trying to win. it's also unfair to pos 3 4 5. Could make role queue more valuable, but still unfair if some ppl only play as support etc. Also why not just make a dummy account and die on purpose if matched into same game as enemy?

My suggestion would be to make rank more fun like guild/hero contracts or achievements. e.g. Last hit/deny tower +1 mmr. Negate fatal damage to an ally with healing within 1 seconds +1 mmr. Godlike killing streaks. same hero win streaks. 3 win streak +30 (60 mmr), 6 wins +60 (90 mmr) so smurfs get to their proper MMR faster. Also fun to try keep win streaks and so on.

Feeder should not lose mmr. Multiple reports from different parties should so doesn't matter true / false.
e.g. 1 report -1mmr. 2 or more reports from different parties -5 mmr
